juniper_official/Solutions/VPLS-view/vpls-view.playbook

```diff
@@ -0,0 +1,5 @@
+iceberg {
+    playbook vpls-view {
```

Comment:
Could you please add comments detailing about the playbook functionality. Please look at below example.

https://github.com/Juniper/healthbot-rules/blob/master/juniper_official/Linecard/linecard-kpis.playbook

This comment is valid for rules as well